pytorch_pfn_extras/handler.py

```diff
@@ -205,6 +206,9 @@ def __init__(self, logic, entry_runtime, options):
                     Default is an empty list.
                 * ``'async'`` (bool):
                     If ``True``, async mode is enabled. Default is ``False``.
+                * ``'sync_trigger'`` (tuple or Trigger):
+                    Trigger used to synchronize iterations in
+                    asynchronous mode. Default is ``None``.
```

Comment:
Do we need to pass the trigger manually? I was thinking that `_synchronize_train` is automatically called for all `needs_model_state = True` extensions.